src/assets/i18n/fr.json5

```diff
@@ -3358,10 +3826,10 @@
   "person.page.birthdate": "Date de naissance",
 
   // "person.page.edit": "Edit this item",
-  "person.page.edit": "Éditer cet item",
+  "person.page.edit": "Éditer cet enregistrement",
```

Comment:
This seems to be the only place where the term Item is translated. Maybe better to keep Item for now.